query-node/manifest.yml

```
@@ -170,20 +161,20 @@ typegen:
     - referendum.VoteRevealed
     - referendum.StakeReleased
     # Bounty
-    - bounty.BountyCreated
-    - bounty.BountyCanceled
-    - bounty.BountyVetoed
-    - bounty.BountyFunded
-    - bounty.BountyMaxFundingReached
-    - bounty.BountyFundingWithdrawal
-    - bounty.BountyCreatorCherryWithdrawal
-    - bounty.BountyRemoved
-    - bounty.WorkEntryAnnounced
-    - bounty.WorkEntryWithdrawn
-    - bounty.WorkEntrySlashed
-    - bounty.WorkSubmitted
-    - bounty.OracleJudgmentSubmitted
-    - bounty.WorkEntrantFundsWithdrawn
+    # - bounty.BountyCreated
```

Let's add a TODO note here
```suggestion
    # TODO: uncomment this + event handlers below after bounties are repaired
    # - bounty.BountyCreated
```